extensions/metrics/src/observers/metrics.push.observer.ts

```diff
@@ -26,7 +26,7 @@ export class MetricsPushObserver implements LifeCycleObserver {
     this.gateway = new Pushgateway(gwConfig.url);
     this.interval = setInterval(() => {
       this.gateway.pushAdd({jobName: 'loopback'}, () => {});
-    }, gwConfig.interval || 5000);
+    }, gwConfig.interval ?? 5000);
```

Comment:
This change will allow `interval` to be set to value `0`. What is the expected behavior when the interval is `0`? Should we release this a semver-major?

Reply:
See https://nodejs.org/api/timers.html#timers_setinterval_callback_delay_args
> When delay is larger than 2147483647 or less than 1, the delay will be set to 1. Non-integer delays are truncated to an integer.